src/PHPCR/Shell/Serializer/NodeNormalizer.php

```diff
@@ -85,15 +85,15 @@ public function normalize($node, $format = null, array $context = [])
     /**
      * {@inheritdoc}
      */
-    public function supportsNormalization($data, $format = null)
+    public function supportsNormalization($data, $format = null, array $context = []): bool
```

Comment:
technical a bc break but I think we could ignore it?

Reply:
this is to adjust to symfony 7 BC breaks, right?

it would only be an issue if somebody extended the NodeNormalizer class. for this component imo that is not expected, agree that we don't do a new major version for that change.